src/models/game-model.js

```diff
@@ -0,0 +1,34 @@
+import { MAX_RANDOM_NUMBER, MIN_RANDOM_NUMBER } from "../constants.js";
+
+export function getRandomNumber(minNumber, maxNumber) {
+    return Math.floor(Math.random() * maxNumber) + minNumber;
+}
+
+export function getUpAndDownStatus({ randomNumber, userInput }) {
+    if (randomNumber > Number(userInput)) {
+        return 'up';
+    }
+
+    if (randomNumber < Number(userInput)) {
+        return 'down'
+    }
+
+    if (isNaN(userInput)) {
+        return 'error'
+    }
+
+    return 'correct';
+}
+
+
+export function isValidUserInput({ userInput }) {
```

Comment:
이 경우에는 요렇게 받아주는게 좋다고 생각합니다~
```suggestion
export function isValidUserInput(userInput) {
```